lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3215,6 +3229,7 @@ def setUp(self):
 
         self.instructor = InstructorFactory(course_key=course.id)
         self.client.login(username=self.instructor.username, password='test')
+        self.grant_sudo_access(self.course.id.to_deprecated_string(), 'test')
 
     def test_change_due_date(self):
         url = reverse('change_due_date', kwargs={'course_id': self.course.id.to_deprecated_string()})
```

Comment:
It would be good to add some tests that verify that the api endpoints aren't available without sudo mode.